packages/affiliates/gas-rebate/VoterGasRebateV2.ts

```diff
@@ -6,14 +6,35 @@
 
 import { getAddress } from "@uma/contracts-node";
 import hre from "hardhat";
-const { ethers } = hre as any;
-import { BigNumber } from "ethers";
+import { BigNumber, ethers } from "ethers";
```

Comment:
is this same version as hre.ethers? I recall we had interface conflicts before due to importing ethers from different places.